packages/fast-usdc/src/fast-usdc.start.js

```diff
@@ -0,0 +1,257 @@
+import { AmountMath } from '@agoric/ertp/src/amountMath.js';
+import { deeplyFulfilledObject, makeTracer, objectMap } from '@agoric/internal';
+import { Fail } from '@endo/errors';
+import { E } from '@endo/far';
+import { M } from '@endo/patterns';
+import { makeMarshal } from '@endo/marshal';
+import { FastUSDCTermsShape } from './type-guards.js';
+import { fromExternalConfig } from './utils/config-marshal.js';
+
+/**
+ * @import {DepositFacet} from '@agoric/ertp/src/types.js'
+ * @import {TypedPattern} from '@agoric/internal'
+ * @import {Instance, StartParams} from '@agoric/zoe/src/zoeService/utils'
+ * @import {Board} from '@agoric/vats'
+ * @import {ManifestBundleRef} from '@agoric/deploy-script-support/src/externalTypes.js'
+ * @import {BootstrapManifest} from '@agoric/vats/src/core/lib-boot.js'
+ * @import {LegibleCapData} from './utils/config-marshal.js'
+ * @import {FastUsdcSF, FastUsdcTerms} from './fast-usdc.contract.js'
+ */
+
+const trace = makeTracer('FUSD-Start', true);
+
+const contractName = 'fastUSDC';
```

Comment:
the initialism isn't relevant at this level.
```suggestion
const contractName = 'fastUsdc';
```

Reply:
looks odd to my eye. but ok.